ts/data.ts

```diff
@@ -1 +1,20 @@
 /* exported data */
+interface Data {
+  favorites: Exercise[];
+}
+
+let fitlogData: Data = {
+  favorites: [],
+};
+
+window.addEventListener('beforeunload', () => {
+  const dataJSON = JSON.stringify(fitlogData);
+  localStorage.setItem('fitlog-data', dataJSON);
+});
+
+const previousJSON = localStorage.getItem('fitlog-data');
+if (previousJSON) {
+  const parsedDataJSON = JSON.parse(previousJSON);
+  fitlogData = parsedDataJSON;
+  console.log(fitlogData);
+}
```

Comment:
`console.log`. Even if you are going to use this in the next feature to check data, you should remove it because otherwise users who visit the application can see data being console logged